db/silkworm/common/worker.hpp

```diff
@@ -30,30 +30,32 @@ class Worker {
 
     Worker() = default;
 
+    /* Not moveable / copyable */
     Worker(Worker const&) = delete;
     Worker& operator=(Worker const&) = delete;
 
     virtual ~Worker();
 
-    void start();                  // Start worker thread
+    void start(bool wait = true);  // Start worker thread (by default waits for status)
     void stop(bool wait = false);  // Stops worker thread (optionally wait for complete stop)
     void kick();                   // Kicks worker thread if waiting
 
     // Whether or not this worker/thread should stop
-    bool should_stop() { return state_.load() == WorkerState::kStopping; }
+    bool should_stop() { return state_.load(std::memory_order_relaxed) == WorkerState::kStopping; }
```

Comment:
I'd avoid non-default (i.e. non `memory_order_seq_cst`) memory order since getting it right might be tricky, while performance gain is probably miniscule.

Reply:
Yes ... was a leftover. Removed everywhere else.